tests/cp_mock.py

```diff
@@ -1,6 +1,13 @@
+import importlib
 import sys
 
+# Add necessary paths
 sys.path.append("emulator/cp/")
 sys.path.append("flight/")
+
+# Mock other modules
 sys.modules["micropython"] = __import__("micropython_mock")
```

Comment:
would rather you put these two sys.modules calls in emulator/cp_mock.py

Reply:
Will keep the 2 cp_mock separate for now. We need to fix some emulator interfacing/path issues with the unit test pipeline first